### PR TITLE
Warn about using deprecated cmake property.

### PR DESCRIPTION
**Pull request by alexlarsson**

### CODE REVIEW

File: src/builder-module.c [hunk 1]
```diff
@@ -341,6 +341,8 @@ builder_module_set_property (GObject      *object,
 
     case PROP_CMAKE:
       self->cmake = g_value_get_boolean (value);
+      if (self->cmake)
+        g_printerr ("The cmake module property is deprecated, use buildsystem cmake instead.\n");
```
Comment (TingPing) [inline src/builder-module.c]: Maybe mention `cmake-ninja` also?

Comment (alexlarsson) [reply src/builder-module.c]: I dunno how important that is for the warning.

Comment (TingPing) [reply src/builder-module.c]: Well it just avoids wasting everybodies time IMO:

- User sees new warning, follows warning and uses `buildsystem: cmake`
- Somebody else points out `cmake-ninja` exists now and is just better
- User updates it a second time

Comment (alexlarsson) [reply src/builder-module.c]: Changing to cmake-ninja is not just a deprecation avoidance though, but could cause build issues. I guess its worth mentioning though.

**Comment by alexlarsson** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**